index.html

```diff
@@ -207,15 +262,17 @@ <h2  class="contactme" >Contact me </h2>
         </div>
             <div class="form-group">
                  <input class="texform" type="text"   id="Name"    classe="form-control" placeholder="Enter Your Name" minlength="4" maxlength="30" required />
+            </div>
+            <div class="form-group">          
                 <input class="texform"  type="email"  id="email"  name="email"   placeholder="Enter Your Email"  required  />
-            </div >
+            </div>
             <div class="form-group2">
-                <textarea class="comment"  name="comment-section" id="comment"  maxlength="500"s  placeholder="Write your message here..."  required ></textarea>
+                <input class="texform2"  name="comment"  id="comment"   cols="50"   rows="6"  placeholder="Write your message here..."  required  />
             </div>
-              
-            <div class="buttonsee2">
+            <div class="buttonsee2" id="contactbuttonid">
```

Comment:
Please make sure you are following the Figma template for paddings, font-family, font-size, font color for the input element in the form. 
Figma template:
<img width="180" alt="Screenshot 2022-12-16 at 10 22 51 PM" src="https://user-images.githubusercontent.com/25887253/208173384-50574d24-42a2-4eb3-a7d0-7fd8a7ce6987.png">
Your design:
<img width="736" alt="Screenshot 2022-12-16 at 10 01 41 PM" src="https://user-images.githubusercontent.com/25887253/208173196-87af0f57-7dc3-4e29-9c77-bbdb0dbb9c40.png">